src/contrib/coh_gamma/COHNCGamma_ext/NCgamma_Parameters_GeV.h

```diff
@@ -0,0 +1,74 @@
+#ifndef _PARAMETERS_H_
+#define _PARAMETERS_H_
+#include <cmath>
+#include <complex>
+
+namespace NCgamma_Param
+{
+
+    const std::complex<double> c_i(0.0,1.0);
```

Comment:
private values of constants can lead to inconsistencies .... most of these should already be defined elsewhere